src/utils.ts

```diff
@@ -23,7 +23,7 @@ export const defaultRESTHeaders = {
 
 export function getApolloClient(providerName = 'default'): ApolloClient<unknown> {
   // eslint-disable-next-line
-  const makeApolloClients = require('./graphql/apolloClient');
+  const makeApolloClients = require('./graphql/makeApolloClients');
```

Comment:
can we use `import` instead? Early loading is not the issue anymore, because we use a factory function.